registers/TUTORIAL.md

```diff
@@ -0,0 +1,21 @@
+This tutorial will show you how to get started with Trillian, build a
+quick working demo, and then transform that into a verifiable,
+reliable service.
+
+Step 1
+------
```

Comment:
Consider instead using hashes for headers (ex. `## Step 1`)

Reply:
Why?

Reply:
I thought you needed to for Markdown to turn it into a header, but turns out this works too.